unicodetools/org/unicode/text/UCD/MakeUnicodeFiles.txt

```diff
@@ -1,6 +1,6 @@
 Generate: .
-CopyrightYear: 2020
-DeltaVersion: 17
+CopyrightYear: 2021
+DeltaVersion: 1
```

Comment:
We usually set this one higher than the highest dsomething that is published, so that there is no collision. Looks like we have a d7 now, so this wants to be 8 so that your generated files say d8.